src/rust/runtime/memory/demibuffer.rs

```diff
@@ -77,12 +88,12 @@ use ::std::{
 #[repr(C)]
 #[repr(align(64))]
```

Comment:
Since we are not matching DPDK's structure we may want to remove this and let the compiler handle alignments.